automation/vars/default.yaml

```diff
@@ -0,0 +1,46 @@
+---
+vas:
+  hci:
+    repo: https://github.com/openstack-k8s-operators/architecture.git
+    branch: main
+    stages:
+    - path: examples/va/hci/control-plane/nncp
+      validation: "oc wait nncp --for jsonpath='{.status.conditions[0].reason}'=SuccessfullyConfigured --timeout=60" 
+      values:
+      - name: network-values
+        src_file: network_values.yaml
+        dest_file: values.yaml
+
+    - path: examples/va/hci/control-plane
+      validation: "oc wait osctlplane controlplane --for condition=Ready --timeout=600s"
+      values:
+      - name: network-values
+        src_file: network_values.yaml
```

Comment:
that "src_file" value is pointing to a non-existing file in the architecture tree. IIRC it was pointing to a static file in Andrew's PoC.

Would it be possible to ensure `src_file` points to the correct source file we have to modify?
Doing so, we'd be able to:
- copy that original file if we don't have done it already (ensure we're able to consume a "vanilla" original file)
- run the ci_gen_kustomize_values against it
- ensure we output the updated file to the `src_file` (override it)

Reply:
We could do that and then also get rid of `dest_file`, if the idea is just to write the `src_file` back to the same location.

Reply:
`dest_file` would then be consumed in order to know "where" to output the `kustomize build` result

Reply:
Well, it wouldn't matter where you put the `kustomize build` result.  The resulting output of that is completely self-contained, so you could place wherever you wish and just `oc apply -f` it.

Reply:
Yes, that's a fair point